ansible/tasks/hashbang/templates/usr/local/share/man/man7/hashbang.7.j2

```diff
@@ -326,9 +314,6 @@ management.
 .SS Misc. / Unsorted (Sort these!)
 pv - Monitors the progress of data through a pipe.
 
-tsung - Used to stress test HTTP, WebDAV, LDAP, MySQL, PostgreSQL, SOAP, and
```

Comment:
TIL of tsung. Why don't we just *install* this?

Reply:
Okay, added it to packages.